LandSandBoat/server
# [Survival guide] Cleanup, add destination check and move to teleports folder

**_I affirm:_**

## What does this pull request do?

- Cleanup global file.
- Add a check to ensure we have the destination guide unlocked.
- Move both global files to teleports folder.

## Steps to test these changes

Use survival guides. No changes should be observed, unless you try to teleport to a guide you havent unlocked.
